src/comments.md

```diff
@@ -1,5 +1,24 @@
 # Comments
 
+> **<sup>Lexer</sup>**  
+> LINE_COMMENT :  
+> &nbsp;&nbsp; `//` ~[\n\r]*  
+>  
+> BLOCK_COMMENT :  
+> &nbsp;&nbsp; `/*` (BLOCK_COMMENT | .)* `*/`  
+>  
+> OUTER_DOC_LINE_COMMENT :  
+> &nbsp;&nbsp; `//!` ~[\n\r]*  
```

Comment:
Only one of LF/CRLF/EOF can end outer/inner doc line comments and bare CR isn't allowed here. For example, `"//! foo \r//! bar\n"` will lead to a syntax error. The difficulty here would be to express the fact that `//! foo \r//! bar\n` isn't a non-doc comment neither.